pulumi/src/cluster/sts/backup-cron.ts

```diff
@@ -0,0 +1,102 @@
+import { ServiceConfig, StsDefinition } from "../..";
```

Comment:
might as well put this next to `statefulService.ts`, we can take a look together at re organizing things after we get this body of work in